commercial/app/model/commercial/events/LiveEvent.scala

```diff
@@ -28,24 +24,16 @@ case class LiveEvent(eventId: String,
 
 object LiveEvent {
 
-  def apply(event: EBEvent, maybeEventImage: Option[LiveEventImage] = None): LiveEvent = {
-
-    val maybeImageUrl =
-      maybeEventImage match {
-        case Some(eventImage) => Some(eventImage.mainImageUrl)
-        case None => None
-      }
-
+  def apply(event: EBEvent, maybeEventInternalFeed: Option[LiveEventMembershipInfo] = None): LiveEvent =
     new LiveEvent(
       eventId = event.id,
       name = event.name,
       date = event.startDate,
-      eventUrl = event.url,
+      eventUrl = maybeEventInternalFeed map {_.url} getOrElse event.url,
       description = event.description,
       status = event.status,
       venue = event.venue,
       tickets = event.tickets,
-      imageUrl = maybeImageUrl
+      imageUrl = maybeEventInternalFeed map {_.mainImageUrl}
```

Comment:
Good point - images are no longer optional; when parsing our EB feed, if we do not find an associated image from the membership feed, we don't add the LiveEvent to our agent.